src/helpers/data.py

```diff
@@ -0,0 +1,159 @@
+from collections import defaultdict
+import csv
+from datetime import datetime
+import os
+import requests
+
+import constants
+
+
+def fetch_csv(url, as_dicts=False):
+  download = requests.get(url)
+
+  decoded_content = download.content.decode('utf-8')
+
+  return list((csv.DictReader if as_dicts else csv.reader)(decoded_content.splitlines(), delimiter=','))
```

Comment:
I'd separate this into two separate calls:
```
reader = csv.DictReader if as_dicts else csv.reader
return reader(decoded_content.splitlines(), delimiter=',')
```

Reply:
Done